### PR TITLE
Support setting email/login/email for registries in tag_and_push plugin

### PR DESCRIPTION
**Pull request by vrutkovs**
Required change to enable https://github.com/projectatomic/atomic-reactor/pull/466

This also enables setting several secrets for a plugin

* [x] Add tests

### CODE REVIEW

**Comment by mmilata** [comment]
LGTM, wanna add some tests?

**Comment by vrutkovs** [comment]
Right, I added a couple for atomic-reactor, I'm gonna add some here as well

File: osbs/build/build_request.py [hunk 1]
```diff
@@ -200,12 +200,16 @@ def render_tag_and_push_registries(self):
...
-                for registry in self.spec.registry_uris.value:
+                for i, registry in enumerate(self.spec.registry_uris.value):
                     if not registry.uri:
                         continue
 
                     regdict = registries[placeholder].copy()
                     regdict['version'] = registry.version
+                    registry_values = self.spec.registry_secrets.value
+                    if isinstance(registry_values, list) and i < len(registry_values):
+                        regdict['secret'] = os.path.join(SECRETS_PATH, registry_values[i])
```
Comment (lcarva) [inline osbs/build/build_request.py]: Might be worth using `zip_longest` (`izip_longest` in python2):

```python
In [2]: registries = ['one', 'two']

In [3]: secrets = ['shhh']

In [4]: from itertools import izip_longest

In [5]: for registry, secret in izip_longest(registries, secrets):
   ...:     print registry, secret
   ...:     
one shhh
two None
```

```python
registry_values = self.spec.registry_secrets.value or []
```

**Comment by lcarva** [comment]
Small suggestion, otherwise LGTM.

**Comment by vrutkovs** [comment]
Closing in favor of #443

**Pull request closed without merging**